src/clr-angular/forms/styles/_form.clarity.scss

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+.clr-form {
```

Comment:
Wow. Really appreciate you separating the styles into their individual files. Thank you!

BTW each file should use the exports mixin. Eg: https://github.com/vmware/clarity/blob/master/src/clr-angular/button/_buttons.clarity.scss#L92

Reply:
done